.github/workflows/ci.yml

```diff
@@ -64,27 +64,16 @@ jobs:
       # Install dependencies
       - name: Install rush
         run: npm install -g @microsoft/rush
-      - name: Install dependencies
-        run: rush install --max-install-attempts 3
-      # Switch flavor if necessary
-      - name: Switch flavor for build
-        id: switch-flavor
+      # Force build flavor if necessary
```

Comment:
Just an idea, maybe we can create a new command specifically for CI and run it instead of separate [Force build flavor, Install dependencies] steps

Reply:
I like this idea, more reusable and would encapsulate the comments to the individual commands run well as well